<!DOCTYPE html>
<html lang="zh-CN">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution | Xjgou&#39;s blog</title>
    <meta name="generator" content="VuePress 1.9.9">
    <link rel="icon" href="/img/MN.jpg">
    <meta name="description" content=" ">
    <meta name="keywords" content="前端博客,个人技术博客,前端,前端开发,前端框架,web前端,前端面试题,技术文档,学习,面试,JavaScript,js,ES6,TypeScript,vue,python,css3,html5,Node,git,github,markdown">
    <meta name="baidu-site-verification" content="7F55weZDDc">
    <meta name="theme-color" content="#11a8cd">
    
    <link rel="preload" href="/assets/css/0.styles.f7afbda9.css" as="style"><link rel="preload" href="/assets/js/app.31543a42.js" as="script"><link rel="preload" href="/assets/js/2.311a5bd7.js" as="script"><link rel="preload" href="/assets/js/22.e694a6b4.js" as="script"><link rel="prefetch" href="/assets/js/10.915c82a6.js"><link rel="prefetch" href="/assets/js/11.28e2f3cc.js"><link rel="prefetch" href="/assets/js/12.aef53514.js"><link rel="prefetch" href="/assets/js/13.1827c212.js"><link rel="prefetch" href="/assets/js/14.e3d4145f.js"><link rel="prefetch" href="/assets/js/15.bc8ad31f.js"><link rel="prefetch" href="/assets/js/16.c036541f.js"><link rel="prefetch" href="/assets/js/17.0a238010.js"><link rel="prefetch" href="/assets/js/18.2de6b661.js"><link rel="prefetch" href="/assets/js/19.128fc34b.js"><link rel="prefetch" href="/assets/js/20.bd897770.js"><link rel="prefetch" href="/assets/js/21.3e4bf892.js"><link rel="prefetch" href="/assets/js/23.28b96b67.js"><link rel="prefetch" href="/assets/js/24.b934355b.js"><link rel="prefetch" href="/assets/js/25.5fd59300.js"><link rel="prefetch" href="/assets/js/26.fcc2b96d.js"><link rel="prefetch" href="/assets/js/27.772fa486.js"><link rel="prefetch" href="/assets/js/28.f2583550.js"><link rel="prefetch" href="/assets/js/29.cba8d112.js"><link rel="prefetch" href="/assets/js/3.36e21275.js"><link rel="prefetch" href="/assets/js/30.36c60c44.js"><link rel="prefetch" href="/assets/js/31.55f9bf6c.js"><link rel="prefetch" href="/assets/js/32.a32032ca.js"><link rel="prefetch" href="/assets/js/33.06df1fd5.js"><link rel="prefetch" href="/assets/js/34.60dcb338.js"><link rel="prefetch" href="/assets/js/35.42279780.js"><link rel="prefetch" href="/assets/js/36.b197e278.js"><link rel="prefetch" href="/assets/js/37.ad0c37ce.js"><link rel="prefetch" href="/assets/js/38.cefff419.js"><link rel="prefetch" href="/assets/js/39.aa3a00ec.js"><link rel="prefetch" href="/assets/js/4.15e6a037.js"><link rel="prefetch" href="/assets/js/40.9fa4a5d7.js"><link rel="prefetch" href="/assets/js/41.ebe7803c.js"><link rel="prefetch" href="/assets/js/42.d9547100.js"><link rel="prefetch" href="/assets/js/43.9606de09.js"><link rel="prefetch" href="/assets/js/44.51021392.js"><link rel="prefetch" href="/assets/js/45.2fcaa40d.js"><link rel="prefetch" href="/assets/js/46.65b904f7.js"><link rel="prefetch" href="/assets/js/47.b5301257.js"><link rel="prefetch" href="/assets/js/48.871b1ef3.js"><link rel="prefetch" href="/assets/js/49.984319a0.js"><link rel="prefetch" href="/assets/js/5.a183a990.js"><link rel="prefetch" href="/assets/js/50.2b90fa06.js"><link rel="prefetch" href="/assets/js/51.789207c8.js"><link rel="prefetch" href="/assets/js/52.5ed96c4e.js"><link rel="prefetch" href="/assets/js/53.a5e222ff.js"><link rel="prefetch" href="/assets/js/54.ed47f009.js"><link rel="prefetch" href="/assets/js/55.852ed88f.js"><link rel="prefetch" href="/assets/js/56.41b0ed84.js"><link rel="prefetch" href="/assets/js/57.6dc9b7d4.js"><link rel="prefetch" href="/assets/js/58.27e63b7f.js"><link rel="prefetch" href="/assets/js/59.906883a5.js"><link rel="prefetch" href="/assets/js/6.e3a30dad.js"><link rel="prefetch" href="/assets/js/60.ed2b1e47.js"><link rel="prefetch" href="/assets/js/61.a10dc273.js"><link rel="prefetch" href="/assets/js/62.4bc8d946.js"><link rel="prefetch" href="/assets/js/63.f20faa81.js"><link rel="prefetch" href="/assets/js/64.a5259659.js"><link rel="prefetch" href="/assets/js/65.22812e1c.js"><link rel="prefetch" href="/assets/js/7.7fe129ad.js"><link rel="prefetch" href="/assets/js/8.87089bfd.js"><link rel="prefetch" href="/assets/js/9.357d7228.js">
    <link rel="stylesheet" href="/assets/css/0.styles.f7afbda9.css">
  </head>
  <body class="theme-mode-light">
    <div id="app" data-server-rendered="true"><div class="theme-container sidebar-open have-rightmenu"><header class="navbar blur"><div title="目录" class="sidebar-button"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img" viewBox="0 0 448 512" class="icon"><path fill="currentColor" d="M436 124H12c-6.627 0-12-5.373-12-12V80c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12z"></path></svg></div> <a href="/" class="home-link router-link-active"><img src="/img/MN.jpg" alt="Xjgou's blog" class="logo"> <span class="site-name can-hide">Xjgou's blog</span></a> <div class="links"><div class="search-box"><input aria-label="Search" autocomplete="off" spellcheck="false" value=""> <!----></div> <nav class="nav-links can-hide"><div class="nav-item"><a href="/" class="nav-link">首页</a></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="文献阅读" class="dropdown-title"><a href="/pages/10127a/" class="link-title">文献阅读</a> <span class="title" style="display:none;">文献阅读</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><h4>汇总</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/2120d1/" class="nav-link">文献汇总</a></li></ul></li><li class="dropdown-item"><h4>分类</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/c7f75f/" class="nav-link">生物信息学</a></li><li class="dropdown-subitem"><a href="/pages/612e84/" class="nav-link">单细胞空间组</a></li><li class="dropdown-subitem"><a href="/pages/b1886e/" class="nav-link">免疫学相关</a></li><li class="dropdown-subitem"><a href="/pages/0bfd97/" class="nav-link">scRNA-scATAC整合</a></li><li class="dropdown-subitem"><a href="/pages/34b3a1/" class="nav-link">计算生物学</a></li></ul></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="生信学习" class="dropdown-title"><a href="/pages/517c9f/" class="link-title">生信学习</a> <span class="title" style="display:none;">生信学习</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><h4>汇总</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/3d46d5/" class="nav-link">生信工具汇总</a></li></ul></li><li class="dropdown-item"><h4>分类</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/c01d86/" class="nav-link">图像处理</a></li><li class="dropdown-subitem"><a href="/pages/75d747/" class="nav-link">单细胞空间组</a></li></ul></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="数据分析" class="dropdown-title"><a href="/technology/" class="link-title">数据分析</a> <span class="title" style="display:none;">数据分析</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><!----> <a href="/pages/9a7ee40fc232253e/" class="nav-link">技术文档</a></li><li class="dropdown-item"><!----> <a href="/pages/4c778760be26d8b3/" class="nav-link">GitHub技巧</a></li><li class="dropdown-item"><!----> <a href="/pages/117708e0af7f0bd9/" class="nav-link">Nodejs</a></li><li class="dropdown-item"><!----> <a href="/pages/41f87d890d0a02af/" class="nav-link">博客搭建</a></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="更多" class="dropdown-title"><a href="/more/" class="link-title">更多</a> <span class="title" style="display:none;">更多</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><!----> <a href="/pages/2d615df9a36a98ed/" class="nav-link">心情杂货</a></li><li class="dropdown-item"><!----> <a href="/pages/baaa02/" class="nav-link">实用技巧</a></li><li class="dropdown-item"><!----> <a href="/friends/" class="nav-link">友情链接</a></li></ul></div></div><div class="nav-item"><a href="/pages/45a884/" class="nav-link">关于</a></div><div class="nav-item"><a href="/pages/beb6c0bd8a66cea6/" class="nav-link">收藏</a></div> <a href="https://github.com/gouxiaojuan/gouxiaojuan.github.io" target="_blank" rel="noopener noreferrer" class="repo-link">
    GitHub
    <span><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewBox="0 0 100 100" width="15" height="15" class="icon outbound"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path> <polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg> <span class="sr-only">(opens new window)</span></span></a></nav></div></header> <div class="sidebar-mask"></div> <div class="sidebar-hover-trigger"></div> <aside class="sidebar" style="display:none;"><div class="blogger"><img src="/img/MN.jpg"> <div class="blogger-info"><h3>Xjgou</h3> <span>生信小菜鸡</span></div></div> <nav class="nav-links"><div class="nav-item"><a href="/" class="nav-link">首页</a></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="文献阅读" class="dropdown-title"><a href="/pages/10127a/" class="link-title">文献阅读</a> <span class="title" style="display:none;">文献阅读</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><h4>汇总</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/2120d1/" class="nav-link">文献汇总</a></li></ul></li><li class="dropdown-item"><h4>分类</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/c7f75f/" class="nav-link">生物信息学</a></li><li class="dropdown-subitem"><a href="/pages/612e84/" class="nav-link">单细胞空间组</a></li><li class="dropdown-subitem"><a href="/pages/b1886e/" class="nav-link">免疫学相关</a></li><li class="dropdown-subitem"><a href="/pages/0bfd97/" class="nav-link">scRNA-scATAC整合</a></li><li class="dropdown-subitem"><a href="/pages/34b3a1/" class="nav-link">计算生物学</a></li></ul></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="生信学习" class="dropdown-title"><a href="/pages/517c9f/" class="link-title">生信学习</a> <span class="title" style="display:none;">生信学习</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><h4>汇总</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/3d46d5/" class="nav-link">生信工具汇总</a></li></ul></li><li class="dropdown-item"><h4>分类</h4> <ul class="dropdown-subitem-wrapper"><li class="dropdown-subitem"><a href="/pages/c01d86/" class="nav-link">图像处理</a></li><li class="dropdown-subitem"><a href="/pages/75d747/" class="nav-link">单细胞空间组</a></li></ul></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="数据分析" class="dropdown-title"><a href="/technology/" class="link-title">数据分析</a> <span class="title" style="display:none;">数据分析</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><!----> <a href="/pages/9a7ee40fc232253e/" class="nav-link">技术文档</a></li><li class="dropdown-item"><!----> <a href="/pages/4c778760be26d8b3/" class="nav-link">GitHub技巧</a></li><li class="dropdown-item"><!----> <a href="/pages/117708e0af7f0bd9/" class="nav-link">Nodejs</a></li><li class="dropdown-item"><!----> <a href="/pages/41f87d890d0a02af/" class="nav-link">博客搭建</a></li></ul></div></div><div class="nav-item"><div class="dropdown-wrapper"><button type="button" aria-label="更多" class="dropdown-title"><a href="/more/" class="link-title">更多</a> <span class="title" style="display:none;">更多</span> <span class="arrow right"></span></button> <ul class="nav-dropdown" style="display:none;"><li class="dropdown-item"><!----> <a href="/pages/2d615df9a36a98ed/" class="nav-link">心情杂货</a></li><li class="dropdown-item"><!----> <a href="/pages/baaa02/" class="nav-link">实用技巧</a></li><li class="dropdown-item"><!----> <a href="/friends/" class="nav-link">友情链接</a></li></ul></div></div><div class="nav-item"><a href="/pages/45a884/" class="nav-link">关于</a></div><div class="nav-item"><a href="/pages/beb6c0bd8a66cea6/" class="nav-link">收藏</a></div> <a href="https://github.com/gouxiaojuan/gouxiaojuan.github.io" target="_blank" rel="noopener noreferrer" class="repo-link">
    GitHub
    <span><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewBox="0 0 100 100" width="15" height="15" class="icon outbound"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path> <polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg> <span class="sr-only">(opens new window)</span></span></a></nav>  <ul class="sidebar-links"><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>生物信息学</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading open"><span>单细胞空间组</span> <span class="arrow down"></span></p> <ul class="sidebar-links sidebar-group-items"><li><a href="/pages/1b45a3/" class="sidebar-link">Advances in spatial transcriptomic data analysis</a></li><li><a href="/pages/e49ae7/" aria-current="page" class="active sidebar-link">Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution</a><ul class="sidebar-sub-headers"><li class="sidebar-sub-header level2"><a href="/pages/e49ae7/#abstract" class="sidebar-link">Abstract:</a></li><li class="sidebar-sub-header level2"><a href="/pages/e49ae7/#results" class="sidebar-link">Results:</a><ul class="sidebar-sub-headers"><li class="sidebar-sub-header level3"><a href="/pages/e49ae7/#figure-1-benchmarking-workflow-and-summary-characteristics-of-the-examined-paired-datasets" class="sidebar-link">Figure 1: Benchmarking workflow and summary characteristics of the examined paired datasets.</a></li><li class="sidebar-sub-header level3"><a href="/pages/e49ae7/#figure-2-comparing-the-accuracy-of-eight-integration-methods-capable-of-predicting-the-spatial-distribution-of-rna-transcripts" class="sidebar-link">Figure 2: Comparing the accuracy of eight integration methods capable of predicting the spatial distribution of RNA transcripts.</a></li><li class="sidebar-sub-header level3"><a href="/pages/e49ae7/#figure-3-comparing-the-accuracy-of-the-eight-integration-methods-for-sparse-spatial-expression-matrices-down-sampled-from-the-original-datasets-using-splatter" class="sidebar-link">Figure 3: Comparing the accuracy of the eight integration methods for sparse spatial expression matrices down-sampled from the original datasets using Splatter.</a></li><li class="sidebar-sub-header level3"><a href="/pages/e49ae7/#figure-4-comparing-the-performance-of-the-12-integration-methods-capable-of-deconvoluting-cell-types-of-each-histological-spot" class="sidebar-link">Figure 4: Comparing the performance of the 12 integration methods capable of deconvoluting cell types of each histological spot.</a></li></ul></li></ul></li><li><a href="/pages/d69341/" class="sidebar-link">SODB facilitates comprehensive exploration of spatial omics data</a></li><li><a href="/pages/c733df/" class="sidebar-link">Significance analysis for clustering with single-cell RNA-sequencing data</a></li><li><a href="/pages/824737/" class="sidebar-link">Integration of spatial and single-cell data across modalities with weakly linked features</a></li><li><a href="/pages/58ba19/" class="sidebar-link">A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain</a></li><li><a href="/pages/7065f3/" class="sidebar-link">Benchmarking spatial clustering methods with spatially resolved transcriptomics data</a></li></ul></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>免疫学相关</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>scRNA-scATAC整合</span> <span class="arrow right"></span></p> <!----></section></li><li><section class="sidebar-group collapsable depth-0"><p class="sidebar-heading"><span>计算生物学</span> <span class="arrow right"></span></p> <!----></section></li></ul> </aside> <div><main class="page"><div class="theme-vdoing-wrapper "><div class="articleInfo-wrap" data-v-91157402><div class="articleInfo" data-v-91157402><ul class="breadcrumbs" data-v-91157402><li data-v-91157402><a href="/" title="首页" class="iconfont icon-home router-link-active" data-v-91157402></a></li> <li data-v-91157402><a href="/pages/10127a/#文献阅读" data-v-91157402>文献阅读</a></li><li data-v-91157402><a href="/pages/10127a/#单细胞空间组" data-v-91157402>单细胞空间组</a></li></ul> <div class="info" data-v-91157402><div title="作者" class="author iconfont icon-touxiang" data-v-91157402><a href="https://github.com/gouxiaojuan" target="_blank" title="作者" class="beLink" data-v-91157402>gouxiaojuan</a></div> <div title="创建时间" class="date iconfont icon-riqi" data-v-91157402><a href="javascript:;" data-v-91157402>2023-08-21</a></div> <!----></div></div></div> <!----> <div class="content-wrapper"><div class="right-menu-wrapper"><div class="right-menu-margin"><div class="right-menu-title">目录</div> <div class="right-menu-content"></div></div></div> <h1><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAAAXNSR0IArs4c6QAABGpJREFUSA3tVVtoXFUU3fvOI53UlmCaKIFmwEhsE7QK0ipFEdHEKpXaZGrp15SINsXUWvBDpBgQRKi0+KKoFeJHfZA+ED9KKoIU2gYD9UejTW4rVIzm0VSTziPzuNu1z507dibTTjL4U/DAzLn3nL3X2o91ziX6f9wMFdh6Jvbm9nNSV0msViVO6tN1Rm7NMu2OpeJ9lWBUTDxrJbYTS0hInuwciu9eLHlFxCLCZEk3MegsJmZ5K/JD6t7FkFdEvGUo1g7qJoG3MHImqRIn8/nzY1K9UPKKiJmtnUqHVE3Gbuay6vJE/N2FEmuxFjW2nUuE0yQXRRxLiTUAzs36zhZvOXJPdX850EVnnLZkB8prodQoM5JGj7Xk2mvC7JB8tG04Ef5PiXtG0UtxupRQSfTnBoCy554x18yJHI6I+G5Eru4LHmPJZEQsrvPUbMiA8G/WgMK7w7I+ez7++o2ANfbrjvaOl1tFMs+htG3IrZH9/hDX1Pr8Tc0UvH8tcX29KzAgIGcEkINyW5BF9x891hw6VYqgJHEk0huccS7vh3C6gTiODL+26huuBtbct8eZnqLML8PkxGYpuPZBqtqwkSjgc4mB5gbgig5i+y0UDK35LMxXisn9xQtK+nd26gTIHsHe/oblK/b29fUmN/8Y+9jAQrnBp56m1LcDlDp9irKTExSKduXJVWSqdBMA08pEJnEIOB3FPPMybu/oeV8zFeYN3xx576Q6RH+VmplE4ncQV5v+5rzSoyOU7PuEAg8g803PwBJ0CExno/jcMbN8tONYeOmHiuUNryvm3fRUy4tMPVLdAGkUhNWuggGrJcXPv+ouCjz0MKUHz1J2/E8IC9nqTabcxgaBYM0hPhD5Y65FsbxRQKxCQrDjDctW7PUM3HuZunFyifSAqEfuzCp48Il24luWUWZoyJCaPR82jE0+kFA643wRFVni4RYSq3ohJO2pZ7B5dO4xkDWbEpossJPLSrPjYID8rS2UHTlvyNxqIGsg674XJJ7vnh5L7PNwC4hh2sjCI96mzszOTpxLF0T7l88Yz7lAuK6OnL8gXLOnTvpzSb22YG8W7us3jSebFHeeqnXRG1vt+MoUM84LQIBmMsCTAcOauTh0T0l0neQK7m2bLMt2mGxU3HYssS0J2cdv5wljlPsrIuZLAG/2DOZIXgCYT8uMGZN+e2kSirfxZOPCsC0f24nTZzspnVn9VePS1Z5vubmAGGXG8ZFno9Hel0yfA5ZPhF7Dh972BQJ2qCpgH67lmWtBYbvk6sz02wjky2vXyz0XErP/kFB619js1BtwfOV4OPRqOQBjy3Qbk18vigUPPSD5ceHnwck7W9bhAqZdd7SuG7w4/P2F/GaJh8c7e9qgow+Q7cGBo+98WsLkuktFqiZabtXuQTu/Y5ETbR0v7tNSFnvrmu6pjdoan2KjMu8q/Hmj1EfCO2ZGfEIbIXKUlw8qaX9/b2oeSJmFksSeT/Fn0V3nSypChh4Gjh74ybO9aeZ/AN2dwciu2/MhAAAAAElFTkSuQmCC">Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution<!----></h1>  <div class="theme-vdoing-content content__default"><p>题目: 用于转录分布预测和细胞类型反卷积的空间和单细胞转录组学整合方法的基准测试<br>
DOI: <a href="https://doi.org/10.1038/s41592-022-01480-9" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-022-01480-9<span><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewBox="0 0 100 100" width="15" height="15" class="icon outbound"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path> <polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg> <span class="sr-only">(opens new window)</span></span></a><br>
Cite: Li, B., Zhang, W., Guo, C. et al. Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution. Nat Methods 19, 662–670 (2022).</p> <p><strong>作者介绍：</strong></p> <table><thead><tr><th style="text-align:center;">Kun Qu</th></tr></thead> <tbody><tr><td style="text-align:center;"><img src="http://jcyx.ustc.edu.cn/_upload/article/images/98/e7/b5f2b7524ecc95a13264a819b160/f8d7a581-a9a7-4eab-8e04-36ab469e3dad.png" width="200"></td></tr> <tr><td style="text-align:center;">中国科学技术大学</td></tr> <tr><td style="text-align:center;"><a href="mailto:qukun@ustc.edu.cn">qukun@ustc.edu.cn</a></td></tr></tbody></table> <h2 id="abstract"><a href="#abstract" class="header-anchor">#</a> Abstract:</h2> <p>空间转录组学方法极大地提高了我们检测组织中RNA转录本空间分布的能力，但表征空间中单细胞的全转录组水平数据仍然具有挑战性。
为了满足这一需求，研究人员开发了整合方法，将空间转录组数据与单细胞 RNA-seq 数据相结合，以预测未检测到的转录本的空间分布和/或对组织学切片中的 spot 进行细胞类型反卷积。
然而，迄今为止，还没有独立研究对这些集成方法进行比较分析以对其性能进行基准测试。
在这里，我们使用 45 个配对数据集（包括空间转录组学和 scRNA-seq 数据）和 32 个模拟数据集对 16 种集成方法进行基准测试。
我们发现<span style="color:red;">Tangram、gimVI、SpaGE</span>在预测RNA转录本空间分布方面优于其他集成方法，而<span style="color:red;">Cell2location、SpatialDWLS、RCTD</span>是spot细胞类型反卷积的最佳方法。
我们提供基准管道来帮助研究人员选择最佳的集成方法来处理他们的数据集。</p> <h2 id="results"><a href="#results" class="header-anchor">#</a> Results:</h2> <h3 id="figure-1-benchmarking-workflow-and-summary-characteristics-of-the-examined-paired-datasets"><a href="#figure-1-benchmarking-workflow-and-summary-characteristics-of-the-examined-paired-datasets" class="header-anchor">#</a> Figure 1: Benchmarking workflow and summary characteristics of the examined paired datasets.</h3> <p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig1_HTML.png?as=webp" alt="Figure 1"></p> <div class="custom-block note"><p class="custom-block-title">Figure 1. 基准测试工作流程和所检查的配对数据集的摘要特征</p> <p><strong>(a)</strong> 用于比较配对空间转录组学和 scRNA-seq 数据集的集成方法性能的基准测试工作流程的示意图。我们使用 16 种整合方法来组合空间和单细胞转录组数据，然后比较它们在 (1) 预测 RNA 转录本的空间分布和 (2) 组织学 spot 的细胞类型去卷积方面的性能。我们还评估了集成方法消耗的计算资源。<br> <strong>(b)</strong> 本研究中使用的 45 个配对数据集和 32 个模拟数据集的信息：每个数据集包含同一组织的空间转录组数据和 scRNA-seq 数据。</p></div> <h3 id="figure-2-comparing-the-accuracy-of-eight-integration-methods-capable-of-predicting-the-spatial-distribution-of-rna-transcripts"><a href="#figure-2-comparing-the-accuracy-of-eight-integration-methods-capable-of-predicting-the-spatial-distribution-of-rna-transcripts" class="header-anchor">#</a> Figure 2: Comparing the accuracy of eight integration methods capable of predicting the spatial distribution of RNA transcripts.</h3> <p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig2_HTML.png?as=webp" alt="Figure 2"></p> <div class="custom-block note"><p class="custom-block-title">Figure 2. 比较八种能够预测 RNA 转录本空间分布的整合方法的准确性</p> <p><strong>(a,b)</strong> 数据集 4（seqFISH+；Smart-seq；小鼠皮质）中Igsf21 ( a ) 和Rprm ( b )的空间分布，包括真实值和每种集成方法的预测结果。PCC，真实转录本的表达向量与预测结果的表达向量之间的皮尔逊相关系数。<br> <strong>(c)</strong> 每种集成方法预测数据集4中转录本空间分布的PCC、SSIM、RMSE和JS条形图。数据以平均值±95%置信区间表示。<br> <strong>(d)</strong> 数据集 4 中转录本的 8 种整合方法的 AS 小提琴图(从 PCC、SSIM、RMSE 和 JS 值聚合；参见方法)；n = 4 个基准指标。<br> <strong>(e)</strong> 所有 45 个配对数据集的 8 种整合方法的 AS 箱线图。n = 45 个独立数据集。</p></div> <h3 id="figure-3-comparing-the-accuracy-of-the-eight-integration-methods-for-sparse-spatial-expression-matrices-down-sampled-from-the-original-datasets-using-splatter"><a href="#figure-3-comparing-the-accuracy-of-the-eight-integration-methods-for-sparse-spatial-expression-matrices-down-sampled-from-the-original-datasets-using-splatter" class="header-anchor">#</a> Figure 3: Comparing the accuracy of the eight integration methods for sparse spatial expression matrices down-sampled from the original datasets using Splatter.</h3> <p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig3_HTML.png?as=webp" alt="Figure 3"></p> <div class="custom-block note"><p class="custom-block-title">Figure 3. 比较使用 Splatter 从原始数据集下采样的稀疏空间表达矩阵的八种整合方法的准确性</p> <p><strong>(a)</strong> 根据原始数据和数据集 4 的下采样数据预测的转录本的空间分布的 PCC。对于原始数据和下采样数据，红色转录本的 PCC 值都大于 0.5。红色转录本在所有转录本中的比例定义为RS。<br> <strong>(b)</strong> 8种整合方法以不同下采样率进行下采样在处理数据集 4 的稀疏表达矩阵时的RS值。<br> <strong>(c)</strong> 处理下采样数据集的稀疏表达矩阵时八种整合方法的RS值。n = 19 个独立数据集。</p></div> <h3 id="figure-4-comparing-the-performance-of-the-12-integration-methods-capable-of-deconvoluting-cell-types-of-each-histological-spot"><a href="#figure-4-comparing-the-performance-of-the-12-integration-methods-capable-of-deconvoluting-cell-types-of-each-histological-spot" class="header-anchor">#</a> Figure 4: Comparing the performance of the 12 integration methods capable of deconvoluting cell types of each histological spot.</h3> <p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig4_HTML.png?as=webp" alt="Figure 4"></p> <div class="custom-block note"><p class="custom-block-title">Figure 4. 比较能够对每个组织学点的细胞类型进行解卷积的 12 种整合方法的性能</p> <p><strong>(a)</strong> 数据集 10 的 STARmap 切片（STARmap; Smart-seq; mouse visual cortex），其中细胞按细胞类型注释。每个网格代表一个包含多个单元的模拟点。<br> <strong>(b)</strong> 8种整合方法以不同下采样率进行下采样在处理数据集 4 的稀疏表达矩阵时的RS值。<br> <strong>(c,d)</strong> 从数据集 10 ( c ) 和数据集 4 ( d )模拟的组织学spot的细胞类型组成的AS 条形图。<br> <strong>(e)</strong> 所有 32 个模拟数据集的 10 种集成方法的 AS 箱线图。SpaOTsc 和 novoSpaRc 被排除在外，因为它们需要每个点的空间位置信息，而这在模拟数据集中不可用。</p></div></div></div>  <div class="page-edit"><div class="edit-link"><a href="https://github.com/gouxiaojuan/gouxiaojuan.github.io/edit/master/docs/01.文献阅读/21.单细胞空间组/202205.Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution.md" target="_blank" rel="noopener noreferrer">编辑</a> <span><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewBox="0 0 100 100" width="15" height="15" class="icon outbound"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path> <polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg> <span class="sr-only">(opens new window)</span></span></div> <!----> <!----></div> <div class="page-nav-wapper"><div class="page-nav-centre-wrap"><a href="/pages/1b45a3/" class="page-nav-centre page-nav-centre-prev"><div class="tooltip">Advances in spatial transcriptomic data analysis</div></a> <a href="/pages/d69341/" class="page-nav-centre page-nav-centre-next"><div class="tooltip">SODB facilitates comprehensive exploration of spatial omics data</div></a></div> <div class="page-nav"><p class="inner"><span class="prev">
        ←
        <a href="/pages/1b45a3/" class="prev">Advances in spatial transcriptomic data analysis</a></span> <span class="next"><a href="/pages/d69341/">SODB facilitates comprehensive exploration of spatial omics data</a>→
      </span></p></div></div></div> <div class="article-list"><div class="article-title"><a href="/archives/" class="iconfont icon-bi">最近更新</a></div> <div class="article-wrapper"><dl><dd>01</dd> <dt><a href="/pages/8ebfe8/"><div>
            Motif-based models accurately predict cell type-specific distal regulatory elements
            <!----></div></a> <span class="date">01-20</span></dt></dl><dl><dd>02</dd> <dt><a href="/pages/7065f3/"><div>
            Benchmarking spatial clustering methods with spatially resolved transcriptomics data
            <!----></div></a> <span class="date">03-05</span></dt></dl><dl><dd>03</dd> <dt><a href="/pages/0f3668/"><div>
            Stress-induced red nucleus attenuation induces anxiety-like behavior and lymph node CCL5 secretion
            <!----></div></a> <span class="date">02-20</span></dt></dl> <dl><dd></dd> <dt><a href="/archives/" class="more">更多文章&gt;</a></dt></dl></div></div></main></div> <div class="footer"><div class="icons"><a href="mailto:1027989920@qq.com" title="发邮件" target="_blank" class="iconfont icon-youjian"></a><a href="https://github.com/gouxiaojuan" title="GitHub" target="_blank" class="iconfont icon-github"></a><a href="https://music.163.com/#/playlist?id=755597173" title="听音乐" target="_blank" class="iconfont icon-erji"></a></div> 
  Theme by
  <a href="https://github.com/xugaoyi/vuepress-theme-vdoing" target="_blank" title="本站主题">Vdoing</a> 
    | Copyright © 2023-2026
    <span>xjgou | <a href="https://github.com/xugaoyi/vuepress-theme-vdoing/blob/master/LICENSE" target="_blank">MIT License</a></span></div> <div class="buttons"><div title="返回顶部" class="button blur go-to-top iconfont icon-fanhuidingbu" style="display:none;"></div> <div title="去评论" class="button blur go-to-comment iconfont icon-pinglun" style="display:none;"></div> <div title="主题模式" class="button blur theme-mode-but iconfont icon-zhuti"><ul class="select-box" style="display:none;"><li class="iconfont icon-zidong">
          跟随系统
        </li><li class="iconfont icon-rijianmoshi">
          浅色模式
        </li><li class="iconfont icon-yejianmoshi">
          深色模式
        </li><li class="iconfont icon-yuedu">
          阅读模式
        </li></ul></div></div> <!----> <!----> <!----></div><div class="global-ui"><div></div></div></div>
    <script src="/assets/js/app.31543a42.js" defer></script><script src="/assets/js/2.311a5bd7.js" defer></script><script src="/assets/js/22.e694a6b4.js" defer></script>
  </body>
</html>
